bodhi/server/migrations/versions/cf7257bfa910_add_composed_by_bodhi_flag_to_releases_.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Add composed_by_bodhi flag to releases table.
+
+Revision ID: cf7257bfa910
+Revises: 68f9ccb1f388
+Create Date: 2018-10-20 10:51:03.049052
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'cf7257bfa910'
+down_revision = '68f9ccb1f388'
+
+
+def upgrade():
+    """Add a new composed_by_bodhi column to releases."""
+    op.add_column(
+        'releases', sa.Column('composed_by_bodhi', sa.Boolean(), server_default='t', default=True)
```

Comment:
Hmm, I'd think you could do this to be clearer:

```suggestion
        'releases', sa.Column('composed_by_bodhi', sa.Boolean(), server_default=True, default=True)
```

Reply:
The reason why I choose 't'
https://docs.sqlalchemy.org/en/latest/core/metadata.html#sqlalchemy.schema.Column.params.server_default

Reply:
@sebwoj I'm not sure I follow. Since this is a Boolean field and not a text field, I think it's sensible to use a ```bool``` value.

Reply:
"server_default" docs mentioned earlier points that it should be a "str" type. At first I used "True", but   after running alembic to update my DB, an error was rised. I found solution on StackOverflow("t").